src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStyle = sourceString.split(';');
+  const style = {};
+
+  Object.create(style);
+
+  sourceStyle.forEach((source) => {
+    const data = source.split(':');
+
+    if (data[0].trim()) {
+      const key = data[0].trim();
+      const value = data[1];
+
+      const s = value;
+      let result;
+
+      if (!s.includes(',')) {
```

Comment:
The check for `s.includes(',')` seems unnecessary for this task, as CSS values typically do not contain commas. You can simplify the logic by removing this condition.